README.md

```diff
@@ -65,25 +65,147 @@ with OUT parameters, or sending/retrieving PL/SQL array types - just give a
 
 The array size of the returned PL/SQL arrays can be set with `godror.ArraySize(2000)` (default value is 1024).
 
-## Speed
+## Tuning
```

Comment:
Maybe. What should be grouped together in another documentation? Options?

Reply:
@tgulacsi How is https://github.com/golang/go/tree/master/doc generated- by hand?  I used a single page doc format https://github.com/oracle/node-oracledb/blob/master/doc/api.md for Node.js.  As a reader I like this since I can do quick browser Ctrl-F 'find' page searches.  As a maintainer it means doing a lot of renumbering since, with all the nesting in Node.js, there are a lot of subheadings and numbered sections help a bit to show the indentation level of the heading. Or you could add some doc generation tooling and put doc in multiple files, like https://github.com/oracle/python-cx_Oracle/tree/master/doc.

Reply:
"Tuning" is a good self-contained part, gives itself to separate it.

Reply:
@tgulacsi Let me know what you want to do with it (how you want it separated).  See my other note about doc.

Reply:
Let us know how you want it separated.